lighthouse-core/test/fraggle-rock/scenarios/api-test-pptr.js

```diff
@@ -50,8 +50,7 @@ describe('Fraggle Rock API', () => {
       expect(accessibility.score).toBeLessThan(1);
 
       const {auditResults, erroredAudits, failedAudits} = getAuditsBreakdown(lhr);
-      // TODO(FR-COMPAT): This assertion can be removed when full compatibility is reached.
-      expect(auditResults.length).toMatchInlineSnapshot(`73`);
+      expect(auditResults.map(audit => audit.id)).toMatchSnapshot();
```

Comment:
can we sort these too? just `.sort()` is good.

Reply:
Beware of weak sort